### PR TITLE
add new and update property in chatMessage resource (1196639)

### PR DESCRIPTION
**Pull request by laujan**

### CODE REVIEW

File: api-reference/beta/api/channel-get-message.md [hunk 1]
```diff
@@ -97,7 +97,8 @@ Content-type: application/json
     "etag": "1555375673184",
     "messageType": "message",
     "createdDateTime": "2019-04-16T00:47:53.184Z",
-    "lastModifiedDateTime": null,
+    "lastModifiedDateTime": "2019-02-04T19:58:15.511Z",
```
Comment (RamjotSingh) [inline api-reference/beta/api/channel-get-message.md:100]: lastModifiedTime cant be before created. For examples if lastEdited time is not present u can copy over created date time to keep examples sane.

Comment (RamjotSingh) [reply api-reference/beta/api/channel-get-message.md:100]: Goes for all examples.

File: api-reference/beta/resources/chatmessage.md [hunk 2]
```diff
@@ -48,7 +48,8 @@ Represents an individual chat message within a [channel](channel.md) or [chat](c
 |etag| string | Read-only. Version number of the chat message. |
 |messageType|string|The type of chat message. The possible values are: `message`.|
 |createdDateTime|dateTimeOffset|Read only. Timestamp of when the chat message was created.|
-|lastModifiedDateTime|dateTimeOffset|Read only. Timestamp of when the chat message is created or edited, including when a reply is made (if it's a root chat message in a channel) or a reaction is added or removed. |
+|lastModifiedDateTime|dateTimeOffset|Read only. Timestamp when the chat message is created (initial setting) or edited, including when a reply is made (if it's a root chat message in a channel) or a reaction is added or removed. |
```
Comment (RamjotSingh) [inline api-reference/beta/resources/chatmessage.md:51]: modified might be a better word here you can say modified (including edits)

**Review by RamjotSingh — CHANGES_REQUESTED** [review]
Their should a changelog entry informing people of the change (new property as well changes to existing property). Ideally running stub generator would have done it for you for the new property.

**Comment by RamjotSingh** [comment]
Their should a changelog entry informing people of the change (new property as well changes to existing property). Ideally running stub generator would have done it for you for the new property.

File: api-reference/beta/api/channel-get-message.md (hunk 1 above)
Comment (laujan) [reply api-reference/beta/api/channel-get-message.md:100]: Okay, great catch!

File: api-reference/beta/resources/chatmessage.md [hunk 3]
```diff
@@ -48,7 +48,8 @@ Represents an individual chat message within a [channel](channel.md) or [chat](c
 |etag| string | Read-only. Version number of the chat message. |
 |messageType|string|The type of chat message. The possible values are: `message`.|
 |createdDateTime|dateTimeOffset|Read only. Timestamp of when the chat message was created.|
-|lastModifiedDateTime|dateTimeOffset|Read only. Timestamp of when the chat message is created or edited, including when a reply is made (if it's a root chat message in a channel) or a reaction is added or removed. |
+|lastModifiedDateTime|dateTimeOffset|Read only. Timestamp when the chat message is created (initial setting) or modified, including when a reply is made (if it's a root chat message in a channel) or a reaction is added or removed. |
```
Comment (RamjotSingh) [inline api-reference/beta/resources/chatmessage.md:51]:
```suggestion
|lastModifiedDateTime|dateTimeOffset|Read only. Timestamp when the chat message is created (initial setting) or modified, including when a reaction is added or removed. |
```

File: api-reference/v1.0/resources/chatmessage.md [hunk 4]
```diff
@@ -32,7 +32,8 @@ Represents an individual chat message within a [channel](./channel.md) or (in be
 |etag| string | Read-only. Version number of the chat message. |
 |messageType|string|The type of chat message. The possible values are: `message`.|
 |createdDateTime|dateTimeOffset|Read only. Timestamp of when the chat message was created.|
-|lastModifiedDateTime|dateTimeOffset|Read only. Timestamp of when the chat message is created or edited, including when a reply is made (if it's a root chat message in a channel) or a reaction is added or removed. |
+|lastModifiedDateTime|dateTimeOffset|Read only. Timestamp when the chat message is created (initial setting) or edited, including when a reply is made (if it's a root chat message in a channel) or a reaction is added or removed. |
```
Comment (RamjotSingh) [inline api-reference/v1.0/resources/chatmessage.md:35]:
```suggestion
|lastModifiedDateTime|dateTimeOffset|Read only. Timestamp when the chat message is created (initial setting) or edited, including when a reaction is added or removed. |
```

**Review by RamjotSingh — CHANGES_REQUESTED** [review]

**Review by RamjotSingh — APPROVED** [review]

File: api-reference/beta/api/channel-get-filesfolder.md [hunk 5]
```diff
@@ -91,8 +91,9 @@ Content-length: 401
 {
     "@odata.context": "https://graph.microsoft.com/beta/$metadata#teams('32e3b156-66b2-4135-9aeb-73295a35a55b')/channels('19%3Af253e46c035b42308e9a4a22a87037af%40thread.skype')/filesFolder/$entity",
     "id": "01H7CFEKENJSSIUHGADZBKODARINQC5JMD",
-    "createdDateTime": "0001-01-01T00:00:00Z",
-    "lastModifiedDateTime": "2020-01-23T18:47:13Z",
+    "createdDateTime": "2020-01-01T00:00:00Z",
+    "lastModifiedDateTime": "2020-02-23T18:47:13Z",
```
Comment (RamjotSingh) [inline api-reference/beta/api/channel-get-filesfolder.md:95]: Remove changes.

File: api-reference/beta/api/channel-get-filesfolder.md [hunk 6]
```diff
@@ -102,7 +103,8 @@ Content-length: 401
     },
     "fileSystemInfo": {
         "createdDateTime": "2020-01-23T18:47:12Z",
-        "lastModifiedDateTime": "2020-01-23T18:47:13Z"
+        "lastModifiedDateTime": "2020-02-23T18:47:13Z",
```
Comment (RamjotSingh) [inline api-reference/beta/api/channel-get-filesfolder.md:106]: Remove changes.

File: api-reference/beta/resources/chatmessage.md [hunk 7]
```diff
@@ -53,7 +53,8 @@ Represents an individual chat message within a [channel](channel.md) or [chat](c
 |etag| string | Read-only. Version number of the chat message. |
 |messageType|string|The type of chat message. The possible values are: `message`.|
 |createdDateTime|dateTimeOffset|Read only. Timestamp of when the chat message was created.|
-|lastModifiedDateTime|dateTimeOffset|Read only. Timestamp of when the chat message is created or edited, including when a reply is made (if it's a root chat message in a channel) or a reaction is added or removed. |
+|lastModifiedDateTime|dateTimeOffset|Read only. Timestamp when the chat message is created (initial setting) or modified, including when a reaction is added or removed. |
+|lastEditedDateTime|dateTimeOffset|Read only. Timestamp when edits to the chat message were made. Triggers an "Edited" flag in the Teams UI. If no edits are made the value is `null`.|
```
Comment (RamjotSingh) [inline api-reference/beta/resources/chatmessage.md:57]: Microsoft Teams? For clarifying between Teams as entity collection and Microsoft Teams as a product.

**Review by RamjotSingh — CHANGES_REQUESTED** [review]

File: api-reference/beta/api/channel-get-filesfolder.md [hunk 8]
```diff
@@ -102,7 +103,7 @@ Content-length: 401
     },
     "fileSystemInfo": {
         "createdDateTime": "2020-01-23T18:47:12Z",
-        "lastModifiedDateTime": "2020-01-23T18:47:13Z"
+        "lastModifiedDateTime": "2020-01-23T18:47:13Z",
```
Comment (RamjotSingh) [inline api-reference/beta/api/channel-get-filesfolder.md:106]: Undo this entire file.

File: api-reference/beta/api/channel-get-filesfolder.md [hunk 9]
```diff
@@ -93,6 +93,7 @@ Content-length: 401
     "id": "01H7CFEKENJSSIUHGADZBKODARINQC5JMD",
     "createdDateTime": "0001-01-01T00:00:00Z",
     "lastModifiedDateTime": "2020-01-23T18:47:13Z",
+    "lastEditedDateTime": null,
```
Comment (RamjotSingh) [inline api-reference/beta/api/channel-get-filesfolder.md:96]: Undo.

**Review by RamjotSingh — APPROVED** [review]
Undo the filesFolder files. Rest looks fine.

**Pull request merged**